website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -241,24 +241,37 @@ Once you have the resources ready, you can deploy the Kubeflow manifests for one
 #### [RDS and S3] Deploy both RDS and S3
 
 Use the following command to deploy the Kubeflow manifests for both RDS and S3:
-```sh
-cd $REPO_ROOT  # exported in 1.1 Prerequisites
-while ! kustomize build deployments/rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
-```
+{{< tabpane persistLang=false >}}
+{{< tab header="Kustomize" lang="toml" >}}
+make
```

Comment:
Same as other make related comments

Reply:
@ryansteakley we need to add this, judy just added place holders

We need to change uninstallation as well same as in vanilla deployment guide